### PR TITLE
Testing approach with sql

### PR DESCRIPTION
**Pull request by plooploops**
* Testing approach with SQL using [pytest](https://docs.pytest.org/en/7.1.x/) and [docker container for sql](https://docs.microsoft.com/en-us/sql/linux/sql-server-linux-docker-container-deployment?view=sql-server-ver15&pivots=cs1-bash)
* Incorporate SQL test suite into [vocabulary release pipeline](https://github.com/microsoft/OHDSIonAzure/blob/sql-test-approach/pipelines/README.md#vocabulary-release-pipeline) and as part of github action workflow

* TODO: Investigate data loading as part of SQL test suite

### CODE REVIEW

File: infra/terraform/bootstrap/azure_devops.tf [hunk 1]
```diff
@@ -13,7 +13,7 @@ resource "azuredevops_project" "project" {
   work_item_template = "Agile"
 
   lifecycle {
-    prevent_destroy = true # prevent destroying the project
+    # prevent_destroy = true # prevent destroying the project
```
Comment (yradsmikham) [inline infra/terraform/bootstrap/azure_devops.tf:16]: Is there a reason we are commenting this line out?

File: infra/terraform/bootstrap/azure_devops.tf [hunk 2]
```diff
@@ -33,29 +33,29 @@ resource "azuredevops_git_repository" "repo" {
...
   # Use Terraform import instead, otherwise this resource will destroy the existing repository.
   lifecycle {
-    prevent_destroy = true # prevent destroying the repo
-    ignore_changes = [
-      # Ignore changes to initialization to support importing existing repositories
-      # Given that a repo now exists, either imported into terraform state or created by terraform,
-      # we don't care for the configuration of initialization against the existing resource
-      initialization, # comment this out if you are making a new repo (and want to import from an existing repository)
-    ]
+    # prevent_destroy = true # prevent destroying the repo
```
Comment (yradsmikham) [inline infra/terraform/bootstrap/azure_devops.tf:41]: same here

**Review by yradsmikham — CHANGES_REQUESTED** [review]

File: infra/terraform/bootstrap/azure_devops.tf (hunk 2 above)
Comment (plooploops) [reply infra/terraform/bootstrap/azure_devops.tf]: Good catch, updated!

File: infra/terraform/bootstrap/azure_devops.tf (hunk 1 above)
Comment (plooploops) [reply infra/terraform/bootstrap/azure_devops.tf]: No, this was from testing.  Good catch and updated!

**Review by yradsmikham — APPROVED** [review]

**Pull request merged**